lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -630,13 +698,6 @@ class MockTimers {
     this.#isEnabled = activate;
   }
 
-  /**
-   * Advances the virtual time of MockTimers by the specified duration (in milliseconds).
-   * This method simulates the passage of time and triggers any scheduled timers that are due.
-   * @param {number} [time] - The amount of time (in milliseconds) to advance the virtual time.
-   * @throws {ERR_INVALID_STATE} If MockTimers are not enabled.
-   * @throws {ERR_INVALID_ARG_VALUE} If a negative time value is provided.
-   */
```

Comment:
I'm not sure why these are being removed – unintentional AI action perhaps? Please restore these.